webpack.config.dev.js

```diff
@@ -18,6 +18,19 @@ module.exports = {
                 query: {
                     presets: ['es2015']
                 }
+            },
+            {
+              test: /\.css$/,
+              exclude: /node_modules/,
+              // chained loaders: 
+              // style-loader injects css imported by css-loader
+              loader: 'style-loader!css-loader'
+            },
+            {
+              test: /\.html$/,
+              exclude: /node_modules/,
+              // creates Object for import in index.js
+              loader: 'html-loader?exportAsDefault'
```

Comment:
I did some testing and research and I think we should change this to use the default CommonJS module format. That's what most of bundle.js uses. So the line would just change to:

```
 loader: 'html-loader'
```

We can discuss this more if you want me to explain what I found.